src/components/MiddleSection/index.tsx

```diff
@@ -0,0 +1,21 @@
+import CategorySection from '~components/Category';
+import OnSaleSection from '~components/OnSaleSection';
+import Trending from '~components/Trending';
+
+import styles from './styles.module.scss';
+
+function MiddleSection() {
+    return (
+        <div className={styles.container}>
```

Comment:
Being this the main content container, it is best to use a `<main>` tag instead of a div